src/server/templates/typescript.ts

```diff
@@ -411,7 +411,88 @@ export interface Database {
           }
         }`
     })}
-}`
+}
+
+export type Tables<
```

Comment:
@kiwicopple do you think we need a separate type for `Views`? Or fine to use `Tables` for `Views`?